Gemfile

```diff
@@ -45,7 +45,7 @@ gem 'unsplash'
 
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
-  gem 'byebug'
+  gem 'byebug', '~> 11.0.1'
```

Comment:
Why do we need this constraint? 
I think we have to add constraint if that depends on something. And in that case, we have to add a comment about the cause.

Reply:
I like `~>` kind of constraints for two reasons
- first of all if you run a `bundle update`, it will less probably break, because inly patch version changes are allowed to be done
- second, it makes the developer a bit more conscious, because it forces a bit more to check change-logs before updating a gem.